packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -20,33 +20,88 @@ import {
   useImperativeHandle,
 } from "react";
 import {
-  Box,
+  Box as MuiBox,
   Checkbox as MuiCheckbox,
-  Chip,
+  Chip as MuiChip,
   ListItemSecondaryAction,
   ListSubheader,
-  MenuItem,
+  MenuItem as MuiMenuItem,
   Select as MuiSelect,
+  SelectChangeEvent,
 } from "@mui/material";
 import { SelectProps as MuiSelectProps } from "@mui/material";
 
 import { Field } from "./Field";
 import { FieldComponentProps } from "./FieldComponentProps";
-import { CheckIcon } from "./icons.generated";
+import { CheckIcon, CloseCircleFilledIcon } from "./icons.generated";
 import type { AllowedProps } from "./AllowedProps";
 import {
   ComponentControlledState,
   FocusHandle,
   useInputValues,
   getControlState,
 } from "./inputUtils";
+import styled from "@emotion/styled";
+import {
+  useOdysseyDesignTokens,
+  DesignTokens,
+} from "./OdysseyDesignTokensContext";
 
 export type SelectOption = {
   text: string;
   type?: "heading" | "option";
   value?: string;
 };
 
+const SelectContainer = styled.div`
+  position: relative;
+  width: 100%;
+  display: flex;
+`;
+
+const PlaceholderValuesContainer = styled("div", {
+  shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
+})<{
+  odysseyDesignTokens: DesignTokens;
+}>`
```

Comment:
I don't understand what `shouldForwardProp` is doing here. Can you explain how this works?

Reply:
I have no idea, this came from `main` when I merged in the latest

Reply:
I ended up removing it. In theory — according to the styled docs — we need this to ensure that our `odysseyDesignTokens` prop is passed on to the styles but not the component itself. In practice, it seems we can circumvent it entirely.